clients/go/pkg/auditopt/config.go

```diff
@@ -269,6 +246,15 @@ func labelsFromConfig(cfg *api.Config) audit.Option {
 	return audit.WithMutator(&lp)
 }
 
+func justificationFromConfig(ctx context.Context, cfg *api.Config) (audit.Option, error) {
+	jvsClient, err := client.NewJVSClient(ctx, &client.JVSConfig{JVSEndpoint: cfg.Justification.PublicKeysEndpoint})
+	if err != nil {
+		return nil, err
+	}
+	lp := justification.NewProcessor(jvsClient)
+	return audit.WithMutator(lp), nil
```

Comment:
nit: for such local var, it's ok to use one char var name

```suggestion
	p := justification.NewProcessor(jvsClient)
	return audit.WithMutator(p), nil
```